docs/guide/rest-resources.md

```diff
@@ -244,4 +244,64 @@ will also include the pagination information by the following HTTP headers:
 * `X-Pagination-Per-Page`: The number of resources in each page;
 * `Link`: A set of navigational links allowing client to traverse the resources page by page.
 
+Since collection in REST APIs is a data provider, it shares all data provider features i.e. pagination and sorting.
+
 An example may be found in the [Quick Start](rest-quick-start.md#trying-it-out) section.
+
+### Filtering collections <span id="filtering-collections"></span>
+
+Since version 2.0.13 Yii provides a facility to filter collections. In case you are using `ActiveController` you can enjoy it
+right away. An example can be found in the [Quick Start](rest-quick-start.md#trying-it-out) section. In case you're
+implementing an endpoint yourself, filtering could be done like the following:
+
+```php
+$filter = new ActiveDataFilter([
+    'searchModel' => 'app\models\PostSearch'
+]);
+
+$filterCondition = null;
+
+// You may load filters from any source. For example,
+// if you prefer JSON in request body,
+// use Yii::$app->request->getBodyParams() below:
+if ($filter->load(\Yii::$app->request->get())) { 
```

Comment:
Hm, Interesting realization of `yii\rest\DataFilter::load()`
```php
    /**
     * @inheritdoc
     */
    public function attributes()
    {
        return [
            $this->filterAttributeName
        ];
    }

    /**
     * @inheritdoc
     */
    public function formName()
    {
        return '';
    }
```
@samdark My be we need add link to this in comment?
```php
/**
 * You may load filters from any source. For example
 * if you prefer JSON in request body,
 * use Yii::$app->request->getBodyParams() below:
 * @see \yii\rest\DataFilter::load()
 * @see \yii\rest\DataFilter::attributes()
 * @see \yii\rest\DataFilter::filterAttributeName
 */
```